tests/Resources/wasm.addontest.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2020-Present by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-vars: "off" */
+/* eslint no-unused-expressions: "off" */
+/* eslint max-len: "off" */
+/* eslint no-undef: "off" */
+/* eslint comma-spacing: "off" */
+/* eslint array-bracket-spacing: "off" */
+'use strict';
+const should = require('./utilities/assertions');
+
+describe('WebAssembly', () => {
```

Comment:
According to https://github.com/react-native-community/jsc-android-buildscripts/issues/113#issuecomment-540992345 and https://bugs.webkit.org/show_bug.cgi?id=194257#c71 iOS *does* support WebAssembly, just not on simulators. So we'll need to skip the tests when on iOS sims.